### PR TITLE
RA-1805:Local variables should not be declared and then immediately r…

### PR DESCRIPTION
**Pull request by gracebish**
Changes made
Dropped the local variables which were immediately returned. 
Issue worked on
RA-1805

### CODE REVIEW

**Comment by gracebish** [comment]
@sacull, Kindly review

File: omod/src/main/java/org/openmrs/module/registrationapp/converter/RegistrationSummaryExtensionsGenerator.java [hunk 1]
```diff
@@ -89,9 +89,8 @@ public static boolean isRegAppConfig(AppDescriptor app) {
 	 * Generates a comprehensive registration summary ID from an app ID and section ID.
 	 */
 	public static String generateRegSummaryId(String regAppId, String sectionId) {
-		String regSummaryId = regAppId.substring(0, regAppId.indexOf("registerPatient")) 
+		return (regAppId.substring(0, regAppId.indexOf("registerPatient"))
 								+ "summary."
```
Comment (sacull) [inline omod/src/main/java/org/openmrs/module/registrationapp/converter/RegistrationSummaryExtensionsGenerator.java:93]:
```suggestion
		return (regAppId.substring(0, regAppId.indexOf("registerPatient")) + "summary." + sectionId);
```
Why not in one line?

Ref: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Others
> Long lines. Lines should not exceed 125 characters.

File: omod/src/main/java/org/openmrs/module/registrationapp/converter/RegistrationSummaryExtensionsGenerator.java [hunk 2]
```diff
@@ -89,9 +89,8 @@ public static boolean isRegAppConfig(AppDescriptor app) {
 	 * Generates a comprehensive registration summary ID from an app ID and section ID.
 	 */
 	public static String generateRegSummaryId(String regAppId, String sectionId) {
-		String regSummaryId = regAppId.substring(0, regAppId.indexOf("registerPatient")) 
+		return (regAppId.substring(0, regAppId.indexOf("registerPatient"))
 								+ "summary." 
-								+ sectionId;
-		return regSummaryId;
+		        + sectionId);
```
Comment (sacull) [inline omod/src/main/java/org/openmrs/module/registrationapp/converter/RegistrationSummaryExtensionsGenerator.java:94]:
```suggestion
```
Same as above.

**Comment by sacull** [comment]
The code is fine, but I don't quite understand the PR description:
image

**Comment by HerbertYiga** [comment]
simply improve the ticket  description @gracebish

**Review by HerbertYiga — APPROVED** [review]
@sacull  @dkayiwa   i see this looks good to be merged!

**Pull request merged**